Dockerfile

```diff
@@ -8,6 +8,7 @@ ARG NDCTL_BUILD_DEPS="os-core-dev devpkg-util-linux devpkg-kmod devpkg-json-c"
 #pull dependencies required for downloading and building libndctl
 ARG CACHEBUST
 RUN swupd update && swupd bundle-add ${NDCTL_BUILD_DEPS} go-basic-dev && rm -rf /var/lib/swupd
+RUN ldconfig
```

Comment:
Interesting find :+1: 

Can you add a comment before "RUN ldconfig" along the lines of:
```
# Workaround for "pkg-config: error while loading shared libraries" when using older Docker (see https://github.com/clearlinux/distribution/issues/831).
```

Then we might remember to remove this.